### ISSUE TITLE
Questions for Open NC FAQ

### ISSUE THREAD

**Issue by ejaxon** [Open-NC/leadership-tasks#42]
I am working on an FAQ to add to the Open NC website. Before I actually start attempting answers, I'd like to get agreement on what the questions might be. As part of that, it probably would be helpful to identify _who_ we are targeting. Some that I think are obviously on the list are:

- NC brigade members
- Local & state government workers
- Journalists
- Other regional brigade associations (e.g. the one California is thinking to start)

I've stubbed out the page [here](https://github.com/Open-NC/open-nc.github.io/blob/open-nc-faq/faq.md) with some initial questions that occurred to me.

Please contribute thoughts @ChrisTheDBA, @LauraMB12, @SagarMishra, @jhibbets, @Jillzey.

**Comment by ChrisTheDBA** [comment]
Somehow I have deleted the file. Hold on. Try this link.
<https://github.com/Open-NC/open-nc.github.io/blob/master/faq.md>


Chris Mathews

**Comment by ejaxon** [comment]
Yeah, looks like you deleted the branch I created for it. Here are the questions I had: 

### What is the purpose of the Open NC Collaborative?
TBD

### What is the relationship between Open NC and local NC brigades?
TBD

### What about places that don't have a Code for America brigade?
TBD

**Comment by ejaxon** [comment]
Just keep adding questions here. I'll create a new branch off master when I'm ready to start writing

**Comment by ChrisTheDBA** [comment]
Sorry. When I went to merge the file, it wanted to do a pull request to the
original Code for Pittsburgh site. When I recreated it, I could remember
the exact questions.

Chris Mathews

**Comment by LauraMB12** [comment]
Eric,

I’m glad to edit- when you get to that step.

Thanks!
Laura M. Biediger
Community Engagement Coordinator
Neighborhood Improvement Services, City of Durham
807 E. Main Street
Durham, NC 27701

P 919-560-1647 X34259
F 919-560-1790

Laura.Biediger@Durhamnc.gov<mailto:Laura.Biediger@DurhamNC.gov>
www.DurhamNC.gov<http://www.durhamnc.gov/>

Visit our team’s new website at DurhamCommunityEngagement.org<durhamcommunityengagement.org>.
